cmp/store/StoreFilterField.js

```diff
@@ -39,27 +39,30 @@ export const [StoreFilterField, storeFilterField] = hoistCmp.withFactory({
 
     render({gridModel, store, ...props}) {
         throwIf(gridModel && store, "Cannot specify both 'gridModel' and 'store' props.");
-
         if (!store) {
             gridModel = withDefault(gridModel, useContextModel(GridModel));
-            store = gridModel ? gridModel.store : null;
+            store = gridModel?.store ?? null;
         }
 
-        // Right now we freeze the initial props -- could be more dynamic.
-        const implModel = useLocalModel(() => new StoreFilterFieldImplModel({gridModel, store, ...props}));
-        return XH.isMobile ? mobileStoreFilterFieldImpl({implModel, ...props}) : desktopStoreFilterFieldImpl({implModel, ...props});
+        const model = useLocalModel(() => new StoreFilterFieldImplModel({gridModel, store, ...props}));
+        model.updateFilterProps(props);
+        return XH.isMobile ?
+            mobileStoreFilterFieldImpl({...props, model}) :
+            desktopStoreFilterFieldImpl({...props, model});
     }
 });
 
 StoreFilterField.propTypes = {
 
     /**
-     * Field on optional model to which this component should bind its value. Not required
-     * for filtering functionality (see `gridModel`, `onFilterChange`, and `store` props), but
-     * allows the value of this component to be controlled via an external model observable.
+     * Field on optional model to which this component should bind its value. Specify this
+     * field, or value to control the state of this component.
```

Comment:
Tripped up by "or value?" - is that correct?

Reply:
fixed.